web/templates/homepage/index.html.eex

```diff
@@ -5,8 +5,8 @@
   </div>
 </section>
 
-<section class="pt1 pb3 pt5-l ph4 ph5-m ph6-l lm-bg-dark-blue tc">
-  <p class="white tl">
+<section class="pt1 pb3 ph4 ph5-m ph6-l lm-bg-dark-blue tc">
+  <p class="white tl tc-l">
```

Comment:
Just to confirm, tc-l will only centre the text on large screeen, does the text need to be centre on mobile too?